README.md

```diff
@@ -101,9 +101,18 @@ Many more features can be added by importing an existing JavaScript or CSS frame
 
 ## Installation
 
-### Requirements
+### Running from pre-compiled binaries
 
-You need [Node](https://nodejs.org) installed on your machine to install and run Asciidoctor Web PDF.
+Asciidoctor Web PDF is available for each release as a zip file for 64-bit Windows, Linux and MacOS.
+
+Go to the [releases](https://github.com/Mogztter/asciidoctor-web-pdf/releases) page and download the appropriate binary for your platform.
+Extract this to a folder of your choosing.
+
+From a command line in this folder you can then run the `asciidoctor-web-pdf` command.
```

Comment:
Did you check that the binary is executable? I think permissions (+x) are lost while zipping?

Reply:
I did check by running for both Linux and Windows. I don't think any permissions are lost: 

```
19:09 $ unzip -Z asciidoctor-web-pdf-linux.zip | grep asciidoctor-web-pdf
Archive:  asciidoctor-web-pdf-linux.zip
-rwxrwxr-x  3.0 unx 119469939 bx defN 21-Jan-06 12:46 asciidoctor-web-pdf
```
```
19:11 $ unzip -Z asciidoctor-web-pdf-win.zip | grep asciidoctor-web-pdf
Archive:  asciidoctor-web-pdf-win.zip
-rwxrwxr-x  3.0 unx 120793115 bx defN 21-Jan-06 12:46 asciidoctor-web-pdf.exe
```